mteb/tasks/Retrieval/multilingual/XQuADRetrieval.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+
+from hashlib import sha256
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import MultilingualTask
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+_LANGUAGES = {
+    "ar": ["arb-Arab"],
+    "de": ["deu-Latn"],
+    "el": ["ell-Grek"],
+    "en": ["eng-Latn"],
+    "es": ["spa-Latn"],
+    "hi": ["hin-Deva"],
+    "ro": ["ron-Latn"],
+    "ru": ["rus-Cyrl"],
+    "th": ["tha-Thai"],
+    "tr": ["tur-Latn"],
+    "vi": ["vie-Latn"],
+    "zh": ["zho-Hans"],
+}
+
+
+class XQuADRetrieval(MultilingualTask, AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="XQuADRetrieval",
+        dataset={
+            "path": "xquad",
+            "revision": "51adfef1c1287aab1d2d91b5bead9bcfb9c68583",
+        },
+        description="XQuAD is a benchmark dataset for evaluating cross-lingual question answering performance. It is repurposed retrieving relevant context for each question.",
+        reference="https://huggingface.co/datasets/xquad",
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["validation"],
+        eval_langs=_LANGUAGES,
+        main_score="ndcg_at_10",
+        date=("2019-05-21", "2019-11-21"),
+        form=["written"],
+        domains=["Web"],
+        task_subtypes=[],
```

Comment:
Any guess about this field? Mayeb you can check the paper?

Reply:
This dataset was proposed for question task so I have updated the metadata accordingly.